### DISCUSSION TITLE
About security update policy

### DISCUSSION THREAD

**Discussion by minty99** [microsoft/onnxruntime#13602] [Other Q&A]
I'm trying to add support for previous versions of onnxruntime in `brew`: https://github.com/Homebrew/homebrew-core/pull/115298

The maintainers wants to know security update policy of onnxruntime project.

> Upstream should have a release branch for each formula version, and have an explicit policy of releasing security updates for each version when necessary.

Does ORT project have explicit policy for security updates?

**Comment by snnn** [comment]
It is listed at https://github.com/microsoft/onnxruntime/blob/main/SECURITY.md 

And we only patch the last release.
